### PR TITLE
Build/paths

### PR DESCRIPTION
**Pull request by FrozenKiwi**
We have been using a patchwork of path management techniques to manage our monorepo.

Now tooling support is catching up, we can remove our various patches and just use the standardized methods.

This will allow us to simplify the last remaining manual step in our publishing process.

NOTE: Part of this process required updating Webpack to 5 (and all associated plugins).

Tested:
 - Full rebuilds, run dev on site-app/site-landing/broker-service

### CODE REVIEW

File: apps/site-app/src/app.tsx [hunk 1]
```diff
@@ -26,9 +26,6 @@ import { App } from 'containers/App';
 // Import Language Provider
 import { LanguageProvider, Languages } from '@thecointech/shared/containers/LanguageProvider';
 
-// Load the favicon
-import '!file-loader?name=[name].[ext]!./images/favicon.ico';
```
Comment (FrozenKiwi) [inline apps/site-app/src/app.tsx:30]: favicon is managed directly by webpack

File: apps/site-app/src/app.tsx [hunk 2]
```diff
@@ -67,10 +64,3 @@ if (module.hot) {
 }
 
 render(translations);
-
-// Install ServiceWorker and AppCache in the end since
-// it's not most important operation and if main code fails,
-// we do not want it installed
-if (process.env.NODE_ENV === 'production') {
-  require('offline-plugin/runtime').install();
-}
```
Comment (FrozenKiwi) [inline apps/site-app/src/app.tsx:76]: offline-plugin is not compatible with Webpack5, but at this point we aren't using it anyway

File: package.json [hunk 3]
```diff
@@ -109,10 +112,10 @@
     "@types/redux-devtools": "3.0.47",
     "@types/testing-library__jest-dom": "^5.9.5",
     "@types/url-join": "^4.0.0",
-    "@types/webpack": "4.39.0",
-    "@types/webpack-env": "^1.15.1",
+    "@types/webpack": "5.28.0",
```
Comment (FrozenKiwi) [inline package.json:115]: moved webpack plugins to base, as they are shared by 4 projects now

File: tsconfig.base.json [hunk 4]
```diff
@@ -36,26 +36,7 @@
     "downlevelIteration": true,
 
     "typeRoots": ["./node_modules/@types"],
-    "plugins": [{ "name": "typescript-plugin-css-modules" }],
-    "baseUrl": "./any/random/path",
-    "paths": {
-      "@thecointech/firestore/*":   ["../../../libs/firestore/build/*"],
-      "@thecointech/broker-db/*":   ["../../../libs/broker-db/build/*"],
```
Comment (FrozenKiwi) [inline tsconfig.base.json:43]: Getting rid of this makes me feel good :-)

**Review by MarieTheCoin — APPROVED** [review]

**Pull request merged**